### PR TITLE
docs: updates to docker readme

### PR DESCRIPTION
**Pull request by towerhand**
Minor change, please let me know if you would like me to add or change anything.

### CODE REVIEW

**Comment by towerhand** [comment]
@zwave-js-bot fix lint

File: docker/README.md [hunk 1]
```diff
@@ -16,13 +16,19 @@ Supported architectures are:
 - `armv7` (Ex. Raspberry PI)
 - `arm64` (Ex. OrangePI NanoPI)
 
+Supported tags are:
+- `latest` for the newest official release.
+- `master` most recent fixes and merged PRs.
```
Comment (robertsLando) [inline docker/README.md:21]: There are also `sha-<commit-sha>` and `<version>` tags (like `2.1.0`)

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 1 above)
Comment (towerhand) [reply docker/README.md:21]: Ok, will add that after work. How often are you planning to cut the master tag? I could add that too.

Comment (robertsLando) [reply docker/README.md:21]: > How often are you planning to cut the master tag
What do you mean?

Comment (towerhand) [reply docker/README.md:21]: If I’m not mistaken, with zwavejs/zwavejs2mqtt:latest the image gets built when there is a release, so for the master tag, does it get built daily? Just thinking to highlight the benefits of it.

Comment (robertsLando) [reply docker/README.md:21]: It's built on every push to `master` branch

Comment (towerhand) [reply docker/README.md]: Please check again when you get a chance. I think it should be good now.

**Review by robertsLando — APPROVED** [review]

**Pull request merged**